server/src/index.jsx

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const app = express();
+const port = 3002;
+const bodyParser = require('body-parser');
+const db = require('../../database/index.js');
+
+app.get('/', (req, res) => res.send('CavaTable reviews module server is working!'));
+
+app.use(bodyParser.json());
+
+app.listen(port, () => console.log(`CavaTable is listening on port ${port}`));
```

Comment:
app.listen statement should be the last line in your express server. It goes after your routes.